### PR TITLE
OP-162 Various improvements/formatting/etc.

### PR DESCRIPTION
**Pull request by dbmalkovsky**

### CODE REVIEW

File: src/main/java/org/isf/opd/gui/OpdBrowser.java [hunk 1]
```diff
@@ -107,8 +108,9 @@
 	private static final long serialVersionUID = 2372745781159245861L;
 
 	private static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("dd/MM/yy");
-	private final int defaultPageSize = 100;
-	
+	private static final int defaultPageSize = 100;
```
Comment (mwithi) [inline src/main/java/org/isf/opd/gui/OpdBrowser.java:111]: uppercase

File: src/main/java/org/isf/utils/jobjects/PaginatedTableDecoratorFull.java [hunk 2]
```diff
@@ -61,172 +60,169 @@
...
-    private void init() {
-        initDataModel();
-        initPaginationComponents();
-        initListeners();
-        paginate();
-    }
+	private void initDataModel() {
+		TableModel model = table.getModel();
+		if (!(model instanceof PageableTableModel)) {
+			throw new IllegalArgumentException("TableModel must be a subclass of ObjectTableModel");
```
Comment (mwithi) [inline src/main/java/org/isf/utils/jobjects/PaginatedTableDecoratorFull.java:108]:
```suggestion
			throw new IllegalArgumentException("TableModel must be a subclass of PageableTableModel");
```

File: src/main/java/org/isf/utils/jobjects/PaginatedTableDecoratorSimple.java [hunk 3]
```diff
@@ -14,137 +36,134 @@
...
+					paginate();
+				}
+			});
+		}
+	}
+
+	private void initDataModel() {
+		TableModel model = table.getModel();
+		if (!(model instanceof PageableTableModel)) {
+			throw new IllegalArgumentException("TableModel must be a subclass of ObjectTableModel");
```
Comment (mwithi) [inline src/main/java/org/isf/utils/jobjects/PaginatedTableDecoratorSimple.java:102]:
```suggestion
			throw new IllegalArgumentException("TableModel must be a subclass of PageableTableModel");
```

File: src/main/java/org/isf/opd/gui/OpdBrowser.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/opd/gui/OpdBrowser.java:111]: done

**Pull request merged**